src/frontends/tensorflow/src/translate_session.cpp

```diff
@@ -433,23 +462,24 @@ void TranslateSession::translate_graph(const ov::frontend::InputModel::Ptr& inpu
         }
     }
 
-    if (saved_model_inputs.get() && saved_model_inputs->size() > 0) {
-        for (auto param : params) {
-            // If parameter isn't found in a map of known inputs - mark them as unused
-            // and try to remove it in the normalize step later
-            if (!apply_saved_model_names(param, saved_model_inputs)) {
-                param->get_output_tensor(0).add_names({"saved_model_unused"});
-            }
+    if (saved_model_inputs || saved_model_outputs) {
```

Comment:
I'm not sure I understand how you exclude MetaGraph.

Could you explain?

Reply:
All Metagraph models I observe do not have the signature and this functionality is not tested and was not covered before. Did you test it on your own?

Reply:
As we discussed - no known models, and no proves it couldn't be made, in theory. So, I'm ok with original change.